### PR TITLE
Upgrade callButton

### PR DESCRIPTION
**Pull request by marcoambrosini**
Screenshot 2022-01-25 at 10 12 19
>

Signed-off-by: marco <marcoambrosini@pm.me

### CODE REVIEW

**Comment by nickvergessen** [comment]
Why is the button not primary anymore?

**Comment by marcoambrosini** [comment]
Because given its size I think it will be easy enough to spot without the huge 44x120px blue patch. Don't you think?  And finally we have a usable secondary.

File: src/components/TopBar/CallButton.vue [hunk 1]
```diff
@@ -98,6 +107,8 @@ export default {
 		ActionButton,
 		VideoOff,
 		MenuDown,
+		Button,
+		Video,
```
Comment (nickvergessen) [inline src/components/TopBar/CallButton.vue:111]:
```suggestion
		ActionButton,
		Button,
		Video,
		VideoOff,
		MenuDown,
```

**Comment by nickvergessen** [comment]
The secondary button is pretty broken with dark mode:

### With patch

Theme: `#082321`
Normal | Hover
---|---
Bildschirmfoto von 2022-01-26 14-03-47 | Bildschirmfoto von 2022-01-26 14-03-52

### Before 

Theme: `#082321`
Normal | Hover
---|---
Bildschirmfoto von 2022-01-26 14-10-54 | Bildschirmfoto von 2022-01-26 14-10-57

### With patch

Theme: `#C3E2E3`
Normal | Hover
---|---
Bildschirmfoto von 2022-01-26 14-04-27 | Bildschirmfoto von 2022-01-26 14-04-31
Bildschirmfoto von 2022-01-26 14-05-00 | Bildschirmfoto von 2022-01-26 14-05-06

Similar with some colors in bright mode:
Theme: `#B8D5D6`
Normal | Hover
---|---
Bildschirmfoto von 2022-01-26 14-05-42 | Bildschirmfoto von 2022-01-26 14-05-45


### Before 

In the old version there was at least a border making it a bit more clear it's a clickable area:

Bildschirmfoto von 2022-01-26 14-10-26

---

This is not good enough for the most important button on Talk.

**Comment by marcoambrosini** [comment]
closing in favor of https://github.com/nextcloud/spreed/pull/6963?notification_referrer_id=NT_kwDOAZm9L7MzMjE3ODMxMjQ0OjI2ODUyNjU1#event-6187228030

**Pull request closed without merging**